command/acl/token/update/token_update_test.go

```diff
@@ -122,6 +122,19 @@ func TestTokenUpdateCommand(t *testing.T) {
 		require.Len(t, token.Policies, 1)
 	})
 
+	// update with add-policy-name
+	t.Run("add-policy-name", func(t *testing.T) {
+		token := run(t, []string{
+			"-http-addr=" + a.HTTPAddr(),
+			"-accessor-id=" + token.AccessorID,
+			"-token=root",
+			"-add-policy-name=" + policy.Name,
```

Comment:
These add tests should be testing that `token.Policies` increments by 1, right?

You're adding the same `policy` that was used by the test above so I think this block is currently a no-op.